Dockerfile

```diff
@@ -14,15 +14,8 @@ RUN apk update && \
     python-dev \
     libffi-dev \
     openssl-dev && \
-  pip install --upgrade \
-    pip && \
-  pip install \
-    azure-common>=1.0.0 \
-    requests>=2.9.1 \
-    cryptography>=1.2.2 \
-    azure-servicemanagement-legacy>=0.20.1 \
-    azure-storage>=0.20.3 \
-    blobxfer==0.9.9.11 && \
+  pip install --upgrade pip && \
+  pip install blobxfer && \
```

Comment:
I don't think that we want to drop the version constraints entirely, movies knows if the installed version really works properly with this code